pages/common/samtools.md

```diff
@@ -0,0 +1,36 @@
+# samtools
+
+> Tools for handling high-throughput sequencing (genomics) data.
+> Used for reading/writing/editing/indexing/viewing of data in SAM/BAM/CRAM format.
+
+- Convert a SAM input file (-S) to BAM stream (-b) and save to file:
+
+`samtools view -bS {{input.sam}} > {{output.bam}}`
+
+- Take input from stdin (-) and output a region, including the file header (-h). Prints to stdout:
+
+`{{other_command}} | samtools view -h - chromosome:start-end`
+
+- Sort a file and save using -o (format chosen by output file extension):
+
+`samtools sort {{input}} -o {{output.bam}}`
+
+- Index a sorted BAM file (creates {{sorted_input.bai}}):
+
+`samtools index {{sorted_input}}`
+
+- Print alignment statistics about a file:
+
+`samtools flagstat {{sorted_input}}`
```

Comment:
It's a numerical value assigned to a DNA read when it's aligned to a reference genome. There are several of them and they can be combined. There's a nice tool here to decode them, which should give an idea of what they look like: https://broadinstitute.github.io/picard/explain-flags.html